packages/screens/Launchpad/components/LaunchpadBanner.tsx

```diff
@@ -8,11 +8,13 @@ import { BrandText } from "@/components/BrandText";
 import { SVG } from "@/components/SVG";
 import { SpacerColumn } from "@/components/spacer";
 import { useMaxResolution } from "@/hooks/useMaxResolution";
+import { useScreenSize } from "@/hooks/useScreenSize";
 import { fontSemibold16, fontSemibold28 } from "@/utils/style/fonts";
+import { ScreenSizeBreakpoints } from "@/utils/style/layout";
 
 export const LaunchpadBanner: React.FC = () => {
   const { width } = useMaxResolution();
-  const isSmallScreen = width < 500;
+  const isSmallScreen = useScreenSize() === ScreenSizeBreakpoints.SMALL;
```

Comment:
```suggestion
  const isSmallScreen = width < RESPONSIVE_BREAKPOINT_S;
  
  Could you just use this breakpoint ?
  `isSmallScreen` is not really important here since it's just used to change the font on an approximative breakpoint. No need to be precise, so we should use existing breakpoint to avoid adding complexity.

Reply:
[b55404e](https://github.com/TERITORI/teritori-dapp/pull/1276/commits/b55404ed123b48ed783f20ea4af3e3c08881b8f4)